states/import.go

```diff
@@ -0,0 +1,22 @@
+package states
+
+import (
+	"github.com/zclconf/go-cty/cty"
+)
+
+// ImportedObject represents an object being imported into Terraform with the
+// help of a provider. An ImportedObject is a RemoteObject that has been read
+// by the provider's import handler but hasn't yet been committed to state.
+type ImportedObject struct {
+	ResourceType string
+
+	// Value is the value (of the object type implied by the associated resource
+	// type schema) that represents this remote object in Terraform Language
+	// expressions and is compared with configuration when producing a diff.
+	Value cty.Value
+
+	// Internal corresponds to the field of the same name on RemoteObject,
```

Comment:
field name out of sync

Reply:
Ahh yes, thanks! I didn't end up wiring up any other stuff related to import handling yet, and so I wasn't really paying close attention to this type. I'll fix this docstring up for now, but I expect we'll want to fuss with this some more later once the interactions between this and the import operation become clearer.